src/Resources/Data/Users.js

```diff
@@ -0,0 +1,32 @@
+import { db } from '../../Utilities/Firebase';
+
+let firstDoc = null;
+let lastDoc = null;
+
+export const getChunkOfUsers = async (nextOrPrev, itemsPerPage, setIndex) => {
+    
+    //NOTE: endAt takes the doc itself, not its index.
```

Comment:
What is endAt here? do you mean last doc?

Reply:
It is a function inside the `db.Collection` instance.